pkg/aggregate/functions.go

```diff
@@ -91,18 +92,18 @@ func (a *SumAggregator) Aggregate(t int64, v float64) {
 type SqrAggregator struct{ FloatAggregator }
 
 func (a *SqrAggregator) Aggregate(t int64, v float64) {
-	if !math.IsNaN(v) {
+	if utils.IsDefined(v) {
 		a.val += v * v
```

Comment:
Where would these nonnumerical values come from?

Reply:
User input could contain NaN for example

Reply:
User input... NaN in CSV file for example

Reply:
Sure. But then it could also be Inf or -Inf, and it's incorrect to ignore those.

Reply:
Also, it would be legitimate for us to reject nonnumericals as input.

Reply:
Agree on rejecting invalid input. 
Regarding the +Inf/-Inf I see no reason to threat them as numeric values